core/src/main/java/com/linecorp/armeria/client/RefreshingAddressResolver.java

```diff
@@ -263,6 +263,7 @@ public void close() {
 
     final class CacheEntry {
```

Comment:
Note: It may make more sense that `CacheEntry` is not bound to a specific resolver. Since there may be more changes involved, I'm fine with the current implementation

Reply:
The `resolverClosed` flag is shared with `CacheEntry` so I just couldn't make this class static. Let me try that later if we really need to. 😉